cql2pgjson/src/test/java/org/folio/cql2pgjson/CQL2PgJSONTest.java

```diff
@@ -500,94 +515,96 @@ public void caret(String testcase) {
   })
   public void unicode(String testcase) {
     select(testcase);
+    
+    
     select(testcase.replace("==", "==/ignoreCase/ignoreAccents ")
                    .replace("= ", "= /ignoreCase/ignoreAccents "));
   }
 
   @Test
   @Parameters({
-    "address.city= /respectCase Søvang # Lea Long",
-    "address.city==/respectCase Søvang # Lea Long",
-    "address.city= /respectCase SØvang #",
-    "address.city==/respectCase SØvang #",
-    "address.city= /respectCase Sovang # Lea Long",
-    "address.city==/respectCase Sovang # Lea Long",
-    "address.city= /respectCase SOvang #",
-    "address.city==/respectCase SOvang #",
-    "address.city= /respectCase Sövang # Lea Long",
-    "address.city==/respectCase Sövang # Lea Long",
-    "address.city= /respectCase SÖvang #",
-    "address.city==/respectCase SÖvang #",
+    "address.city=  Søvang # Lea Long",
+    "address.city== Søvang # Lea Long",    
```

Comment:
Can you remove the whitespace at the end of the line?
Can you configure your IDE/editor to trim trailing whitespace as described on https://dev.folio.org/guides/developer-setup/#coding-style ?